research/src/components/select-anatomy.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import './select-anatomy.css'
+
+const SelectAnatomy = () => {
+  return (
+    <host>
+      <part name="button">
+        <slot>Default = Split Button Component</slot>
```

Comment:
Split Button or Select?

Reply:
This is actually something that I want to cover in an issue. This is a slot so technically the content is replaceable. I initially had it as a split button and I already plan to have it as a <button> or we'll need to create new element that has the psuedo elements if the desire is truly there to have the current button design as seperate parts as you're correct that a button and split button are very different paradigms.